src/accuracy.js

```diff
@@ -0,0 +1,78 @@
+import { ResultsGrades, REGRESSION_ERROR_TOLERANCE, } from "./constants.js";
```

Comment:
It would be great to have a comment at the top of each file explaining its general purpose.